doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -4075,7 +4075,7 @@ SoftDevice Controller
 The issues in this section are related to :ref:`nrfxlib:softdevice_controller`.
 In addition to the known issues listed here, see also :ref:`softdevice_controller_limitations` for permanent limitations.
 
-.. rst-class:: v2-8-0
+.. rst-class:: v2-9-0 v2-8-0
 
 DRGN-23776: Sending CIS packets with invalid MIC
```

Comment:
This issue is fixed and not present in v2.9

Reply:
We thought we had fixed it but the same issue occurs sporadically. Could you add 2.9 to the affected version for DRGN-23776 please?